actionlist/app/components/action_item_body.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  actionContainer: {
+    backgroundColor: '#990000',
+    padding: 5,
+    borderRadius: 8,
+    width: 100,
+  },
+  routeContainer: {
+    backgroundColor: '#ffffff',
+    padding: 5,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: '#990000',
+    marginRight: 10,
+    width: 100,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  routeButton: {
+    fontSize: 15,
+    color: '#990000',
+  },
+});
+
+const routeLog = () => ({});
+
+const takeAction = () => ({});
+
+const ActionItemBody = rowData => (
+  <View>
+    <Text>{rowData.processType}</Text>
+    <Text>{rowData.initiator}</Text>
+    <Text>{rowData.lastUpdated}</Text>
+    <Text>{rowData.status}</Text>
+    <Text>{rowData.actions}</Text>
+    <View style={{ flexDirection: 'row', flexWrap: 'wrap' }}>
```

Comment:
Using style object literals is very bad for performance (a new object has to be created every time the `ActionItemBody` rerenders even though the contents of the object will never change). This style object should be extracted.